sections/chla_introduction.md

```diff
@@ -7,8 +7,31 @@ The most important actions to be taken are summarized in this simple cheat sheet
 This short summary allows the reader check under time pressure whether key points are taken into account prior to deployment.
 We recommend reading each chapter in detail to ensure the best quality data.
 
+**Pre-deployment/Deployment**  
+(1) Check that sensors are in good condition, and you have selected the best option for the planned mission.
+- BEST: Do two-point calibration ({numref}two_point_calibration).
```

Comment:
I believe the indentation here is causing the issue. It should be:

```
(1) Check that sensors are in good condition, and you have selected the best option for the planned mission.  
    - BEST: Do two-point calibration ({numref}`two_point_calibration`).  
```

Note the 4 blank spaces, like python!